### DISCUSSION TITLE
grapesjs-preset-webpage does not fully appear

### DISCUSSION THREAD

**Discussion by IbrahimALismail** [GrapesJS/grapesjs#5574] [Q&A]
Hello everyone, I tried to build a web builder to help my clients to build their stores or websites. I faced problem with grapesjs-preset-webpage when I added it not all sections (Basic,Forms, and Extra) are appear that only appear is Basic despite also does not appear all of its components. 
this is my code:
```

import React, { useState, useEffect } from 'react';
import {useParams} from "react-router-dom"
import grapesjs from "grapesjs";
import "./styles/main.scss";
import gspPresetWebpage from "grapesjs-preset-webpage"
const Editor = () => {
    const [editor, setEditor] = useState(null);
    const {pageId} = useParams;
    console.log('pageID :>>', pageId);
    useEffect(() => {
    const editor = grapesjs.init({
      container: "#editor",
      height:"300vh",
      plugins: [gspPresetWebpage],
      pluginsOpts: {
        gspPresetWebpage:{
  
        }
      }
      });
      setEditor(editor)
  
      
    }, []);
  return (
    <div className="App">
    <div id="editor"></div>
  </div>
  )
}

export default Editor

```

and you can see the attachment that show the issue: 
<img width="2560" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/140511696/edbc2553-8c5e-4a84-b397-5685389a86eb">

**Comment by jaspreet-singh-1999** [comment]
include other plugin like grapesjs-blocks-basic, grapesjs-plugin-forms.

**Comment by artf** [comment]
@IbrahimALismail 
<img width="632" alt="Screenshot 2023-12-23 at 19 48 12" src="https://github.com/GrapesJS/grapesjs/assets/11614725/863c41de-10c3-4286-a44e-8783b1da5d4e">
